prod

andriy-fesych opened the pull request:
[DEMO LINK](https://andriy-fesych.github.io/layout_landing-page/)

etojeDenys reviewed (changes requested):
1. disable the scroll when menu is open
<img width="1512" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/70220667/b2379aca-fabe-46ee-983b-0ae5b1a94815">


2. make all fields in the form required
<img width="1512" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/70220667/af12a0ab-d74b-4520-80d3-e98827324908">

3. looks good, keep going

anastasiiavorobiova reviewed (changes requested):
Great start! 
<img width="1714" alt="Screenshot 2023-11-21 at 14 55 11" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/f1443181-1f4e-408c-83f7-9f95e76503a8">
<img width="787" alt="Screenshot 2023-11-21 at 14 56 14" src="https://github.com/mate-academy/layout_landing-page/assets/135232001/368a2122-c9c8-4339-b8c6-9594046edd20">
* Check the sizes of your elements
* The menu should be scrollable if needed

andriy-fesych commented:
> The menu should be scrollable if needed

Fixed. Thank You for mentioning.

> Check the sizes of your elements

Sizes are according to mobile mockup. For up to 744px mobile mockup applied. 
[Here is the link to original mock up](https://www.figma.com/file/NZQAIydtHo5QkINyGLHNcq/BIKE-New-Version?type=design&node-id=0-1&mode=design&t=6vcsrKQje2KE0bNY-0)

andriy-fesych commented:
![image](https://github.com/mate-academy/layout_landing-page/assets/77161886/d862e930-330d-4656-a351-414f80f0c53e)
If You tilt your phone to horizontal orientation, my layout shows exactly how it should look like according to mobile mockup.

etojeDenys reviewed (changes requested):
looks good for mobile, keep going

andriy-fesych commented:
> looks good for mobile, keep going

Can You explain what else You want to be fixed please?

etojeDenys reviewed (changes requested):
1. set the height: 100vh to the header
2. we usually don't use vw for the font-size, 'cause it can looks strange on wide screens
<img width="191" alt="image" src="https://github.com/mate-academy/layout_landing-page/assets/70220667/a43c579b-7baf-4fcf-85f6-699fe569f2fe">
try to fix that

maxim2310 reviewed (changes requested):
Great job overall! Just couple tiny things needs to be fixed before approval)

1. need to make first block on whole height of page(100vh)
2. if you make relative sizes for something need to set max value, because on wide screen it can be so large
3. also menu must be stretch on whole height 
![image](https://github.com/mate-academy/layout_landing-page/assets/96265844/eab82a75-a96c-44f6-83d7-7fdf3049cc53)
4. need to check font weight here 
![image](https://github.com/mate-academy/layout_landing-page/assets/96265844/389eade1-c9ea-4539-b329-3493ea06b01f)
5. dont know if you want that it looks like, on my opinion better to stretch background and make common border 
![image](https://github.com/mate-academy/layout_landing-page/assets/96265844/203c2c44-18db-4579-a6f9-7a0a8ac79825)
6. when i hover on text field its jumping a bit 
![image](https://github.com/mate-academy/layout_landing-page/assets/96265844/010eabd6-a496-4022-93ba-5d4413326bd7)
7. need to set min width for prevent this behavior 
![image](https://github.com/mate-academy/layout_landing-page/assets/96265844/9a517df9-2457-4536-8271-82163e543d01)
8. need to remove autofill styles 
![image](https://github.com/mate-academy/layout_landing-page/assets/96265844/306990ad-374e-4a28-9e80-df1f5b3d8d97)
9. need to add indent here like in design 
![image](https://github.com/mate-academy/layout_landing-page/assets/96265844/ee0502f6-f9de-4e6f-a432-399f66b659fe)

andriy-fesych commented:
> Great job overall! Just couple tiny things needs to be fixed before approval)
> 
> 1. need to make first block on whole height of page(100vh)

done: header height set to 100vh.

> 2. if you make relative sizes for something need to set max value, because on wide screen it can be so large

can You please provide an example of such behaviour with resolution mentioned? Checked on ultrawide, couldn`t find an issue. 

> 3. also menu must be stretch on whole height
>    ![image](https://user-images.githubusercontent.com/96265844/284845712-eab82a75-a96c-44f6-83d7-7fdf3049cc53.png)

Done. Stretched menu (nav) to 100vh. Scroll appears, if content does not fit in 100vh.

> 4. need to check font weight here
>    ![image](https://user-images.githubusercontent.com/96265844/284846674-389eade1-c9ea-4539-b329-3493ea06b01f.png)

Font weight was off indeed👍. Fixed.

> 5. dont know if you want that it looks like, on my opinion better to stretch background and make common border
>    ![image](https://user-images.githubusercontent.com/96265844/284847166-203c2c44-18db-4579-a6f9-7a0a8ac79825.png)

Although, there is no requirement for styling cards, fixed the border as per your recommendation. 

> 6. when i hover on text field its jumping a bit
>    ![image](https://user-images.githubusercontent.com/96265844/284847665-010eabd6-a496-4022-93ba-5d4413326bd7.png)

Noted 👍. Fixed.

> 7. need to set min width for prevent this behavior
>    ![image](https://user-images.githubusercontent.com/96265844/284848023-9a517df9-2457-4536-8271-82163e543d01.png)

Noted 👍. Disabled user resizing option.

> 8. need to remove autofill styles
>    ![image](https://user-images.githubusercontent.com/96265844/284848394-306990ad-374e-4a28-9e80-df1f5b3d8d97.png)

I can`t reproduce this on my PC. Can You tell me which OS and Browser You used for this?

> 9. need to add indent here like in design
>    ![image](https://user-images.githubusercontent.com/96265844/284848938-ee0502f6-f9de-4e6f-a432-399f66b659fe.png)

Added indent.


Feel free to add comments.

IvanFesenko reviewed (approved):
Well done
